tensornetwork/backends/jax/jitted_functions.py

```diff
@@ -55,83 +61,88 @@ def jax_lanczos(matvec, arguments, init, ncv, neig, landelta, reortho):
         falls below `landelta`, iteration is stopped.
       reortho: If `True`, reorthogonalize all krylov vectors at each step.
         This should be used if `neig>1`.
+      precision: jax.lax.Precision type used in jax.numpy.vdot
     Returns:
       jax.numpy.ndarray: Eigenvalues
       list: Eigenvectors
     """
-
+    shape = init.shape
+    dtype = init.dtype
+    # TODO (mganahl): replace with restarted classical gram-schmidt
     def body_modified_gram_schmidt(i, vals):
       vector, krylov_vectors = vals
       v = krylov_vectors[i, :]
-      vector -= jax.numpy.vdot(v, vector) * jax.numpy.reshape(v, vector.shape)
+      vector = vector - jax.numpy.vdot(
+          v, vector, precision=precision) * v
```

Comment:
Do we need the line break?

Reply:
yapf's suggestion, can remove it

Reply:
I mean it's no big deal if it's on purpose